swap_meet/vendor.py

```diff
@@ -1,2 +1,58 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, new_item):
+        self.inventory.append(new_item)
+        return new_item
+
+    def remove(self, item_to_remove):
+        if item_to_remove in self.inventory:
+            removed_item = self.inventory.pop(self.inventory.index(item_to_remove))
+            return removed_item
+        return False
```

Comment:
This works well! We can simplify line 13 by using the `remove` method that is present on lists instead of an `index` then a `pop`.